website/project/views/register.py

```diff
@@ -252,6 +252,13 @@ def _get_or_create_identifiers(node):
         }
 
 
+def osf_admin_change_status_identifier(node, status):
+    if node.get_identifier_value('doi') and node.get_identifier_value('ark'):
```

Comment:
The logic that creates identifiers uses an `or` and does not allow making of identifiers if one or the other exists. 
In theory it should not be possible to have one without the other if I understand ezid. Swapping to an `or` or even just checking one or the other is probably sufficient. This was mostly just to be overly explicit about what you should have access to at the time.

With that info, is there a preference for how to write it?